torch/_torch_docs.py

```diff
@@ -2764,7 +2764,7 @@ def merge_dicts(*dicts):
 Keyword args:
     rounding_mode (str, optional): Type of rounding applied to the result:
 
-        * ``"true"`` - default behavior. Performs no rounding and, if both :attr:`input` and
+        * If not specified, performs no rounding and, if both :attr:`input` and
```

Comment:
The signature on line 2744 needs to be updated to "round_mode=None, out=None". It currently still uses 'true' and there's no comma between the kwargs.

Reply:
Divide signature also needs an update:

https://pytorch.org/docs/master/generated/torch.divide.html?highlight=rounding_mode

And true divide doc does, too: 

https://pytorch.org/docs/master/generated/torch.true_divide.html?highlight=rounding_mode

Reply:
Method docs for div, div_, divide, and divide_ also need an update:

https://pytorch.org/docs/master/tensors.html?highlight=tensor%20div#torch.Tensor.div